### ISSUE TITLE
invalid vsphere_server port in variables examples and docs

### ISSUE THREAD

**Issue by alexeig** [hashicorp/learn-terraform-vsphere#10]
Issue: "cannot connect to vCenter" or a similar error using the default port (`8989`) specified in "example" variable blocks.

`terraform.tfvars.example` and `packer/vars.auto.pkrvars.hcl.example` in this repo both have the following line:
```
vsphere_server     = "127.0.0.1:8989"
```
...where a user is expected to substitute the `127.0.0.1` part with the DNS name or IP of the actual vCenter, and leave the port (`8989`) alone as unlike IPs, it's usually specific to an application or service and is supposed to stay the same. However:
- doing so (leaving port `8989` intact) results in "cannot connect" or a similar error when running `packer build` or `terraform plan` commands
- port `8989` does not appear to be valid, and does not appear to be needed.
- I was unable to find any VMware documents referencing that port. E.g.:
    - "[TCP and UDP ports required to access VMware vCenter Server (52963)](https://kb.vmware.com/s/article/52963)" covering vCenter versions 5.x through 6.0 mentions no such port.
    - "[VMware Ports and Protocols](https://ports.esp.vmware.com/)" covering vCenter (among other products) versions 6.5 through 8.0 - ditto, makes no reference to port `8989`.
- Removing the port altogether resolves the issue.

#### Possible solution

- Remove port `8989` from the relevant code, substituting the following snippet in `terraform.tfvars.example` and `packer/vars.auto.pkrvars.hcl.example`, and relevant docs (e.g. [Manage VMs and snapshots on vSphere - Define Variables](https://developer.hashicorp.com/terraform/tutorials/virtual-machine/vsphere-provider#explore-configuration-and-define-variables)):
```vsphere_server     = "127.0.0.1:8989"```
...with:
```vsphere_server     = "127.0.0.1"```

**Comment by jhbarros** [comment]
Yeah, the 8989 port is only for the VCSIM testing environment and this is considered on the docs. On the vSphere production env, you use the same URL you access the vSphere GUI.
